### PR TITLE
Resolves #1002 - adding autograph config

### LINKED ISSUE #1002: Sync Cloud Services::Operations: Autgraph with AUT
### Bugzilla Product

Cloud Services

### Bugzilla Component

Operations: Autograph

### Jira Project Key

AUT

### Whiteboard Tag

jbi-aut

### Project Contact (Bugzilla user id)

hwine@mozilla.com

### Data Sync Options

- [ ] Component (from the Bug and as specified in config)
- [X] Issue Type (please provide mapping, eg. `enhancement` -> `Story`)
- [ ] Status (please provide mapping, eg. `RESOLVED` -> `Done`)
- [ ] Resolution (please provide mapping, eg. `WONTFIX` -> `Won't do`)
- [ ] Whiteboard tags (as labels)
- [X] Assignee (Bugzilla and Jira user must use the same email address)
- [ ] Field changes (recorded as comments on the issue)

### (Internal) For operators

- [ ] [Request](https://mozilla-hub.atlassian.net/servicedesk/customer/portal/4/group/36/create/172) Jira Automation permissions for project

### PR DESCRIPTION
**Pull request by alexcottner**
Issue #1002 - adding autograph mapping.
@grahamalama - does this look right based on the request?

ToDo:
- [x] Make config changes
- [x] Add new webhook for Cloud Services > Autograph

### CODE REVIEW

File: config/config.prod.yaml [hunk 1]
```diff
@@ -608,3 +608,24 @@
...
+        - add_link_to_jira
+        - maybe_assign_jira_user
+      existing:
+        - update_issue_summary
+        - add_jira_comments_for_changes
+        - maybe_assign_jira_user
+      comment:
+        - create_comment
+    issue_type_map:
+      enhancement: Story
```
Comment (grahamalama) [inline config/config.prod.yaml:631]: @hwine, want to confirm with you that this is the only mapping you want.

The default for example is 
```yaml
task: Task
defect: Bug
```

the default for any unmapped issue type is `Task`.

**Review by grahamalama — COMMENTED** [review]
@hwine from [the issue](https://github.com/mozilla/jira-bugzilla-integration/issues/1002):

> My understanding is that, as soon as this is enabled, I can selectively migrate bugs by adding [jbi-aut] to the whiteboard field in Bugzilla.

That's correct 👍🏻 

> I couldn't spot the documentation of the "default step" configuration, so it's unclear to me what additional mappings are needed.

I added one comment about the `issue_type_map`. Your config is sound - if there are other aspects of the bug you'd like to sync, look through the [available steps](https://github.com/mozilla/jira-bugzilla-integration/blob/main/docs/actions.md#available-steps) (which admittedly aren't always well documented, but I hope are at least well-named enough to hint at what they do)

**Review by grahamalama — APPROVED** [review]

File: config/config.prod.yaml (hunk 1 above)
Comment (hwine) [reply config/config.prod.yaml:631]: I'm not following -- I want the defaults you indicate + the `enhancment: Story` in the config. Does that answer the question?

Comment (grahamalama) [reply config/config.prod.yaml:631]: Yep, it does 👍🏻. The full config will be:

```yaml
      enhancement: Story
      task: Task
      defect: Bug
```

**Pull request merged**